src/main/resources/application.conf

```diff
@@ -111,7 +111,7 @@ vsys {
     minimum-in-memory-diff-blocks = 3600
 
     # Blockchain type. Could be TESTNET | MAINNET | CUSTOM. Default value is TESTNET.
-    type = TESTNET
+    type = MAINNET
```

Comment:
If you change the default chain type to Mainnet, you also need to change the default port in the network to 9921